src/main/java/practice/StreamPractice.java

```diff
@@ -23,7 +34,11 @@ public int findMinEvenNumber(List<String> numbers) {
      * But before that subtract 1 from each element on an odd position (having the odd index).
      */
     public Double getOddNumsAverage(List<Integer> numbers) {
-        return 0D;
+        return IntStream.iterate(0, i -> i < numbers.size(), i -> i + 1)
```

Comment:
your version is ok, but can be done easier
```suggestion
        return IntStream.range(0, numbers.size())
```